### PR TITLE
merge rekar-page-signup into develop

### PR DESCRIPTION
**Pull request by amin-hooshmand**

### CODE REVIEW

File: components/Slider.tsx [hunk 1]
```diff
@@ -1,25 +1,25 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
```
Comment (hazharaziz) [inline components/Slider.tsx:1]: It seems that you have done nothing special in this file except formatting the code. Revert the changes.

File: pages/signUp.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
+import { NextPage } from "next";
+import Image from "next/image";
+import React, { ChangeEvent, useState } from "react";
+import translate from "../public/locales/fa.json";
+import { ToastContainer, toast } from "react-toastify";
+
+const SignUp: NextPage = () => {
+  const [userName, setUserName] = useState<string | number>("");
+  const [password, setPassword] = useState<string>("");
+  const [passwordRepet, setPasswordRepet] = useState<string>("");
```
Comment (hazharaziz) [inline pages/signUp.tsx]: - Typo in `passwordRepet`. Correct it.

File: pages/signUp.tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+  const userNameChangeHandler = (event: ChangeEvent<HTMLInputElement>) => {
+    setUserName(event.target.value);
+  };
+
+  const passwordChangeHandler = (event: ChangeEvent<HTMLInputElement>) => {
+    setPassword(event.target.value);
+  };
+
+  const passwordRepetChangeHandler = (event: ChangeEvent<HTMLInputElement>) => {
```
Comment (hazharaziz) [inline pages/signUp.tsx]: Typo

File: public/locales/en.json [hunk 4]
```diff
@@ -44,5 +44,20 @@
     "fiscalYear": "fiscalYear 1401",
     "select": "select",
     "cancel": "cancle"
+  },
+  "signUp": {
+    "welcome": "Login",
+    "details": "Enter Your User Details",
+    "userName": "User Name",
+    "passwrd": "Password",
```
Comment (hazharaziz) [inline public/locales/en.json]: Typo

**Review by hazharaziz — APPROVED** [review]

**Review by amanc1361 — APPROVED** [review]
